### PR TITLE
RFC: move to PRs

### PR DESCRIPTION
**Pull request by zkat**
This does what it says on the tin. Right now, our RFC process specifies that we should do it through Slack posts, but that kinda sucks. Let's use Github instead!

### CODE REVIEW

**Review by varjmes — APPROVED** [review]

File: _pages/rfc-process.md [hunk 1]
```diff
@@ -25,31 +25,42 @@ An RFC can extend, modify, or alter any part of the WeAllJS community policy, wh
...
+* Pin the link to the PR to `#meta`.
 
 <div class="bordered-divider bordered-bottom"></div>
 
 ## How does review work?
 
-All comments on a particular RFC should be done directly on the document: Slack posts have a small speech bubble on the top right of the document window that can be used to add new comments to the document. This helps preserve any conversation that happened, save questions, etc, which will be protected from the Slack's 10k message limit. As much discussion as possible should be handled in the general context of #meta or private messages, or even private channels, but the final document must be available and visible to all members, and be shared in #meta.
+All comments on a particular RFC should be done directly on the PR: An RFC will not be accepted until reviewers are satisfied with changes. Admins hold ultimate authority over whether or not to accept an RFC, if there's ever conflict about it.
+
+Side discussion on the RFC may happen anywhere, though external discussion is better off in the `#meta` channel for better visibility. Please record any relevant decisions as comments on the PR, when they happen in external discussions.
```
Comment (jenniferlynparsons) [inline _pages/rfc-process.md]: Suggestion for clarity: Discussion on the RFC may happen outside the PR, but is best kept in the WeAllJS slack `#meta` channel for better community visibility.

**Comment by zkat** [comment]
@jenniferlynparsons how's this new wording?

**Comment by kimroen** [comment]
I think this sounds good 👍 

Regarding the process, how would a casual WeAllJS member keep up with these if they wanted to do so? Is the suggested way to subscribe to this repository, and if so - should that be written down somewhere?

**Comment by alferpal** [comment]
@kimroen The change introduced to the "How do I create an RFC?" section states as a part of the new process that one of the steps is pinning the PR to #meta. 

So a casual member only needs to be in #meta to keep up with the changes.

Edit: link https://github.com/WeAllJS/wealljs.github.io/pull/42/files#diff-22de1989139a75c7ae66ff354c0af38bR37

**Comment by kimroen** [comment]
@alferpal You're right, I must've missed it. Thanks!

**Pull request merged**